docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,599 @@
+                    required_field_validator,
+                    or_custom_validator(enum_range_validator(list_connected_ledgers())),
+                ]
+            ),
+            reqs=lambda vs: vs["mode"] == "baker" and vs["key_import_mode"] == "ledger",
+        ),
+        Step(
+            id="secret_key",
+            prompt="Provide either the unencrypted or password-encrypted secret key for your address.",
+            help="The format is 'unencrypted:edsk...' for the unencrypted key, or 'encrypted:edesk...'"
+            "for the encrypted key.",
+            validator=Validator(required_field_validator),
+            reqs=lambda vs: vs["mode"] == "baker"
+            and vs["key_import_mode"] == "secret-key",
+        ),
+        Step(
+            id="json_filepath",
+            prompt="Provide the path to your downloaded faucet JSON file.",
+            help="Download the faucet JSON file from https://faucet.tzalpha.net/.\n"
+            "The file will contain the 'mnemonic' and 'secret' fields.",
+            validator=Validator([required_field_validator, filepath_validator]),
+            reqs=lambda vs: vs["mode"] == "baker" and vs["key_import_mode"] == "json",
+        ),
+    ]
+
+    for step in steps:
+        if not step.reqs(values):
+            continue
+
+        validated = False
+        while not validated:
+            print(step.prompt)
+            step.pprint_options()
+            answer = input("> ").strip()
+
+            if answer.lower() in ["quit", "exit"]:
+                print("Exiting Tezos Baking Wizard")
+                return None
+            elif answer.lower() in ["help", "?"]:
+                print(step.help)
+                print()
+            else:
+                if not answer and step.default is not None:
+                    answer = step.default
+
+                try:
+                    answer = step.validator.validate(answer)
+                except ValueError as e:
+                    print(color("Validation error: " + str(e), "\x1b[1;31m"))
+                else:
+                    validated = True
+                    values[step.id] = answer
+
+    return values
+
+
+class Setup:
+    def __init__(self, config):
+        self.config = config
+
+    # Check if an account with the 'baker' alias already exists, and ask the user
+    # if it can be overwritten.
+    def check_baker_account(self, cmd):
+        output = subprocess.run(shlex.split(cmd), capture_output=True)
+        if output.returncode and re.search(
+            b"The secret_key alias baker already exists.", output.stderr
+        ):
+            value = re.search(b"The current value is (.+)\.", output.stderr).group(1)
+            address = subprocess.run(
+                shlex.split("sudo -u tezos tezos-client show address baker"),
+                capture_output=True,
+            )
+            if address.returncode == 0:
+                address = re.search(b"Hash: (.+)\\n", address.stdout).group(1)
```

Comment:
Note, something similar could be done for all regex parsing places, I think

Reply:
FWIW I don't think changing this match here is going to bring us much or indeed that the wording is going to change soon. But it does make sense, so sure.

However, what do you mean by other places? There are four regex searches in the wizard code:
1) one already parses ledger urls which aren't likely to change
2) one is this one (already covered)
3) one ("The secret_key alias baker already exists.") is used to distinguish between different failing modes when importing keys, so we're going to need to provide an up-to-date output like that anyway (we could rely on return codes, but I believe that'd be even less permanent than stderr output).
4) And the last, used in `value`, is supposed to parse both ledger urls and secret keys, so it's easier to do this using the surrounding text than to assemble a giant regex (nobody would ever be able to read, probably). Alternatively, we can remove the "current value" output completely and only show the address, although I believe at least when using ledger it's much easier to remember the mnemonic url than a Tezos address.
I'm wondering what you think about all of these and whether we should change any.

Reply:
Ah you are right, I should have checked better.
1. For a moment I tough that the `ledger:` was part of the url :facepalm: it's ok as-is indeed
2. _covered_
3. I forgot about that, indeed it should be fine as differentiation (or rather alternatives are similar, as you said).
4. Well, admittedly that would be _one_ big regex, but it's only one of those two options, so it could be the combination of two smaller regex: one that you already have (ledger url) and the other one which would be to add.
    I also think that if not now these could be useful (the one for the secret key, in this case) either way to have.